src/ensembl/src/content/app/browser/drawer/Drawer.scss

```diff
@@ -14,30 +16,38 @@
     text-align: center;
   }
 
-  dd {
-    width: 600px;
+  p {
+    margin: 0px;
   }
 
-  label {
-    float: left;
-    font-weight: $light;
-    text-align: right;
-    width: 150px;
+  .nextLine {
+    display: block;
+    margin-bottom: 10px;
   }
 
-  p {
-    margin: 0 0 20px 0;
+  .clearFix {
+    display: grid;
+    grid-template-columns: 100px 1fr;
+    grid-column-gap: 1em;
+    grid-row-gap: 1em;
+  }
+
+  .label {
+    font-weight: $light;
+    text-align: right;
+    width: 90px;
   }
 
   .details {
-    float: right;
+    display: flex;
     text-align: left;
-    width: 400px;
+    width: 490px;
   }
 
   .mainDetail {
     font-size: 16px;
     font-weight: $bold;
+    margin-top: -3px;
```

Comment:
This looks fishy.
I believe you are trying to fight the incorrect alignment of two lines with different font sizes. I talked to Andrea, and she suggests to reduce the font-size of this element from 16px to 14px.

Reply:
Done, decrease the font size to 14px and alignment is fixed but I found out why the alignment was messed up, its because of the line-height.  In the future if we do want bigger font-size we need to check the line height.